### PR TITLE
KAA-972 Sync Nix changes

### PR DESCRIPTION
**Pull request by rasendubi**
/cc @forGGe

### CODE REVIEW

**Comment by RostakaGmfun** [comment]
Run build

File: client/client-multi/client-c/nix/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    esp8266-rtos-sdk = callPackage ./esp8266-rtos-sdk { };
+
+    cc3200-sdk = callPackage ./cc3200-sdk { };
+
+    raspberrypi-tools = callPackage ./raspberrypi-tools { };
+
+    raspberrypi-openssl = callPackage ./raspberrypi-openssl { };
+
+    # Currently, it causes compilation failure, so we use 4.7 for now.
```
Comment (forGGe) [inline client/client-multi/client-c/nix/default.nix:35]: Let's create ticket for that and mentioned here in comments

**Comment by forGGe** [comment]
Reviewed

**Comment by rasendubi** [comment]
http://jira.kaaproject.org/browse/KAA-995 created

**Comment by forGGe** [comment]
+

**Pull request merged**